examples/undocumented/libshogun/regression_lars_observable_path.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+* Written (W) 2019 Giovanni De Toni
+*/
+
+#include <shogun/features/DenseFeatures.h>
+#include <shogun/lib/SGMatrix.h>
+#include <shogun/regression/LeastAngleRegression.h>
+#include <shogun/labels/RegressionLabels.h>
+#include <shogun/mathematics/linalg/LinalgNamespace.h>
+#include <shogun/preprocessor/PruneVarSubMean.h>
+#include <shogun/preprocessor/NormOne.h>
+#include <shogun/lib/parameter_observers/ParameterObserverLogger.h>
+#include <shogun/base/init.h>
+#include <shogun/lib/common.h>
+
+using namespace shogun;
+
+// Data generator
+void generate_data_n_greater_d(SGMatrix<float64_t> &data, SGVector<float64_t> &lab)
```

Comment:
couldnt we use some environment for regression? Then we'd have less code here

Reply:
ah sorry this is libshogun example. What about actually loading regression data from a file?

Reply:
Yes, it could be done. Actually, this example was taken directly from one of the unit tests, so it was not using any data files.